pkg/apis/trustmanager/v1alpha2/types_cluster_bundle.go

```diff
@@ -238,10 +244,12 @@ const (
 type TargetMetadata struct {
 	// Annotations is a key value map to be copied to the target.
 	// +optional
+	// +kubebuilder:validation:XValidation:rule="self.all(k, !k.startsWith('trust-manager.io/'))", reason=FieldValueForbidden, message="must not use forbidden domains as prefixes (e.g., trust-manager.io)"
```

Comment:
question: Does this mean that if we add trust-manager specific annotations or labels, we'd need to list them all in these comments here?

Reply:
Yes, if we want to allow certain annotations/labels with the `trust-manager.io/`-prefix **to trust-manager CRD(s)**, we need to add exclude rules here. FYI: This validation is currently enforced by our validating webhook, so really no change in behavior.

Reply:
Yeah fair enough - it's definitely not a problem, I was just curious :)